internal/driver/driver_test.go

```diff
@@ -227,9 +218,13 @@ func addFlags(f *testFlags, flags []string) {
 	}
 }
 
+func testSourceURL(port int) string {
+	return fmt.Sprintf("http://%s:%d/", testSourceAddress, port)
```

Comment:
consider using `net.JoinHostPort` here; this bites people in subtle ways. (previous comment was eaten by golint commit).